linked-list.yml

```diff
@@ -1,4 +1,4 @@
 ---
-blurb: "Write a Singly-Linked List implementation that uses the Proxy pattern"
+blurb: "Implement a doubly linked list"
 source: "Inspired by 'Data Structures and Algorithms with Object-Oriented Design Patterns in Ruby', singly linked-lists."
```

Comment:
Let's also change the source to "Classic computer science topic" or something, so that it doesn't reference singly-linked list here.